packages/manager/modules/pci/src/projects/project/translations/Messages_es_ES.json

```diff
@@ -15,5 +15,6 @@
   "pci_projects_project_welcome": "Bienvenido a la nueva interfaz de Public Cloud",
   "pci_projects_project_keystone_upgrade": "El 23 de junio de 2020 se producirá un cambio importante. Por favor, planifique su migración a <strong>Keystone v3</strong>.",
   "pci_projects_project_keystone_upgrade_more": "Más información",
-  "pci_projects_project_quota_warning_message": "Por motivos de seguridad, su proyecto tiene un límite. Para aumentar este límite, acceda a la página <a data-ui-sref=\"{{ quotaUrl }}\">Quota and Localisation</a>."
+  "pci_projects_project_quota_warning_message": "Por motivos de seguridad, su proyecto tiene un límite. Para aumentar este límite, acceda a la página <a data-ui-sref=\"{{ quotaUrl }}\">Quota and Localisation</a>.",
+  "pci_projects_maintenance_warning_message": "In order to improve stability of Public Cloud APIs, multiple maintenance will take place in the different Public Cloud regions. Operations will last for a maxim    um of 2 hours in non-working hours. During the maintenance period, it will not be possible in the concerned region to create, modify or delete Cloud resources. <br />You can find details zone by zone on this travaux task : <a href=\"http://travaux.ovh.net/?do=details&id=47661\" target=\"_blank\">http://travaux.ovh.net/?do=details&id=47661</a>"
```

Comment:
```suggestion
  "pci_projects_maintenance_warning_message": "In order to improve stability of Public Cloud APIs, multiple maintenance will take place in the different Public Cloud regions. Operations will last for a maximum of 2 hours in non-working hours. During the maintenance period, it will not be possible in the concerned region to create, modify or delete Cloud resources. <br />You can find details zone by zone on this travaux task : <a href=\"http://travaux.ovh.net/?do=details&id=47661\" target=\"_blank\">http://travaux.ovh.net/?do=details&id=47661</a>"
```